coordinator/gscoordinator/object_manager.py

```diff
@@ -47,6 +66,19 @@ def __init__(self, key, frontend_endpoint, object_id):
         self.closed = False
 
     def submit(self, message, bindings=None, request_options=None):
+        if request_options is not None and "engine" in request_options:
+            from gremlin_python.driver import request
+
+            rm = request.RequestMessage(
+                processor="gae",
```

Comment:
As you would pass `request_options` like `{'engine': 'gae'}`, I think use `processor=request_options['engine']` would be better.

Reply:
to make it runnable ASAP, it is acceptable, you may add a "FIXME".